ocaml/parsing/builtin_attributes.ml

```diff
@@ -453,31 +453,19 @@ let explicit_arity attrs =
   has_attribute ["ocaml.explicit_arity"; "explicit_arity"] attrs
 
 type jkind_attribute =
-  | Any
-  | Value
-  | Void
   | Immediate64
   | Immediate
-  | Float64
 
 let jkind_attribute_of_string = function
-  | "ocaml.any" | "any" -> Some Any
```

Comment:
Not this line, but please also remove `any`, `void`, `value`, and `float64` (and their `ocaml.` counterparts) from `builtin_attrs`.